setup.py

```diff
@@ -11,37 +11,33 @@
 
 def read(fname):
     fpath = os.path.join(os.path.dirname(__file__), fname)
-    with codecs.open(fpath, 'r', 'utf8') as f:
+    with codecs.open(fpath, "r", "utf8") as f:
```

Comment:
It used to be that we built socorro as a Python package and then installed it from that. We don't do that anymore. I'm pretty sure we can nix this file altogether, but figured I'd look into that later.